Modules/System/Record Link Management/src/RecordLinkImpl.Codeunit.al

```diff
@@ -65,6 +65,18 @@ codeunit 4470 "Record Link Impl."
         Note := BinReader.ReadString();
     end;
 
+    procedure RemoveLinks(RecVariant: Variant)
```

Comment:
Since the facade procedure is public, perhaps you might want to add a check to ensure that the passed argument is a record variant.

Then this might also be something you would want to include in your tests.

Reply:
I think I can throw a very generic error message in this case, because it's not something that users can fix themselves anyway.